build.rs

```diff
@@ -19,7 +19,10 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .build_server(false)
         .type_attribute(".", "#[derive(serde::Serialize, serde::Deserialize)]")
         .compile(
-            &["./skywalking-data-collect-protocol/language-agent/Tracing.proto"],
+            &[
+                "./skywalking-data-collect-protocol/language-agent/Tracing.proto",
+                "./skywalking-data-collect-protocol/logging/Logging.proto",
```

Comment:
This is for further using, right?

Reply:
I think log report doesn't implement in the PR.

Reply:
Yes, is better to restore it and add it in future?

Reply:
It is fine, I just want to confirm this. It is OK to keep.